api/config/config_test.json

```diff
@@ -30,7 +30,10 @@
       0.5,
       0.75
     ],
-    "carahsoftEmail": "workbench-notificatio-aaaaekogch7vg3c6hi4sgyx4le@pmi-engteam.slack.com"
+    "carahsoftEmail": "workbench-notificatio-aaaaekogch7vg3c6hi4sgyx4le@pmi-engteam.slack.com",
+    "freeTierCronUserBatchSize": 5,
```

Comment:
I chose a smaller batch size for test because users have too many workspaces and I wanted to run a smaller batch to avoid timing out. The value can be adjusted in other environments as needed if we see that users have too many workspaces.